### ISSUE TITLE
Custom data for webhooks

### ISSUE THREAD

**Issue by drequivalent** [louislam/uptime-kuma#949]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, New Notification

### 🔖 Feature description

Apprise support of XMPP is godawful. One solution would be to write some simple thing in Go that would listen on a port, take a web request and send a message. Problem is, I need to specify a recipient and maybe some auth secret.

### ✔️ Solution

Allow users to add custom data string to the end of the JSON that is sent out on webhook.

**Comment by caronc** [comment]
XMPP supportshould be a bit more stable in the next release of Apprise: [see here] (https://github.com/caronc/apprise/pull/485)

**Comment by deefdragon** [comment]
I'm working on template notifications (#646), which will hopefully allow for custom webhook data, but that's a bit far off.

**Comment by caronc** [comment]
The latest version of Apprise v0.9.6 should be much more stable.  A XMPP Python Developer who manages `slixxmpp` itself did the commit/fix himself.

```bash
pip install apprise==0.9.6
```

Give it a try and by all means feel free to consider another option here, but I also both encourage and appreciate any tickets opened against Apprise in order to make it better.  I certainly don't want to give you guys crap or useless end points.  If they're broken, I'm more then happy to try to fix them! :slightly_smiling_face:

**Comment by drequivalent** [comment]
Still no worky

```
2021-12-14 19:34:42,349 - DEBUG - Using selector: EpollSelector
2021-12-14 19:34:42,349 - INFO - Notifying 1 service(s) asynchronously.
2021-12-14 19:34:42,351 - DEBUG - Using selector: EpollSelector
2021-12-14 19:34:42,356 - DEBUG - Loaded Plugin: RFC 6120: Stream Feature: STARTTLS
2021-12-14 19:34:42,358 - DEBUG - Loaded Plugin: RFC 6120: Stream Feature: Resource Binding
2021-12-14 19:34:42,359 - DEBUG - Loaded Plugin: RFC 3920: Stream Feature: Start Session
2021-12-14 19:34:42,360 - DEBUG - Loaded Plugin: RFC 6121: Stream Feature: Roster Versioning
2021-12-14 19:34:42,361 - DEBUG - Loaded Plugin: RFC 6121: Stream Feature: Subscription Pre-Approval
2021-12-14 19:34:42,364 - DEBUG - Loaded Plugin: RFC 6120: Stream Feature: SASL
2021-12-14 19:34:42,365 - DEBUG - Loaded Plugin: XEP-0030: Service Discovery
2021-12-14 19:34:42,366 - DEBUG - Loaded Plugin: XEP-0199: XMPP Ping
2021-12-14 19:34:42,366 - DEBUG - Event triggered: connecting
sys:1: RuntimeWarning: coroutine 'XMLStream.run_filters' was never awaited
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
2021-12-14 19:34:42,382 - ERROR - Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<XMLStream.run_filters() running at /usr/local/lib/python3.9/dist-packages/slixmpp/xmlstream/xmlstream.py:1033>>
sys:1: RuntimeWarning: coroutine 'XMLStream._connect_routine' was never awaited
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
2021-12-14 19:34:42,382 - ERROR - Task was destroyed but it is pending!
task: <Task pending name='Task-4' coro=<XMLStream._connect_routine() running at /usr/local/lib/python3.9/dist-packages/slixmpp/xmlstream/xmlstream.py:320>>
```

**Comment by caronc** [comment]
Thank you for also opening [apprise/497](https://github.com/caronc/apprise/issues/497); I pinged the expert.  Hopefully he'll get back to us soon.

**Comment by CommanderStorm** [comment]
@drequivalent 
I think this was resolved by https://github.com/louislam/uptime-kuma/pull/3088 which was merged into the upcoming [`1.23` release](https://github.com/louislam/uptime-kuma/milestone/34)
=> Could you close this issue?
